mappings/net/minecraft/datafixer/fix/ComponentFix.mapping

```diff
@@ -0,0 +1,13 @@
+CLASS net/minecraft/unmapped/C_dxmxprhk net/minecraft/datafixer/fix/ComponentFix
```

Comment:
I feel like this name needs a bit more information

Reply:
Updated this to match Mojang's name of `ItemStackComponentRemainderFix`.